### PR TITLE
ref: swc/core@^1.4.0

### PR DESCRIPTION
**Pull request by JiangWeixian**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -76,15 +76,16 @@ mod test {
         "#
     );
 
-    test!(SYNTAX, runner,
+    test_inline!(SYNTAX, runner,
```
Comment (JiangWeixian) [inline src/lib.rs:79]: Looks like in `@swc/core@^1.4.0` const enum handled by swc, will compile into var. Should we remove this test?

https://play.swc.rs/?version=1.4.12&code=H4sIAAAAAAAAA0vOzysuUUjNK81VcMvPV6jmUgACRx0w5cRVywUAZWdi5iAAAAA%3D&config=H4sIAAAAAAAAA2WPSw7DIAxE9zkF8rrbbnqHHgJRJyLiJ%2BxIRVHuXpxA0qobhJ9nPPY6KAUzGXiotX5rkXQmzGddCZXA%2Bl0JcElIJtvEcOtdJmmN2hHuaDs6wDpPyOJCujc5uBgJu7wxb4Mdy3egiT5lJPoVilSHyeF%2F3P5cg04TWHrG17JbOC8H8520g%2BUoWbOm%2BhhmAhk4bB9kMbooGgEAAA%3D%3D

**Comment by JiangWeixian** [comment]
@hjkcai cc

File: src/lib.rs (hunk 1 above)
Comment (hjkcai) [reply src/lib.rs:79]: I think adding `export` keyword to this test is fine. Just keep it.

**Comment by hjkcai** [comment]
Sorry for the late reply. Thank for your contribution!

**Comment by hjkcai** [comment]
I have published version `0.2.0` including your change. There is an error running the `const_enum` test case. I have fixed that in 9255ba5.

**Pull request merged**